cli/packages/cmd/secrets.go

```diff
@@ -38,12 +38,12 @@ var secretsCmd = &cobra.Command{
 			}
 		}
 
-		infisicalToken, err := util.GetInfisicalServiceToken(cmd)
-
+		token, err := util.GetInfisicalToken(cmd)
 		if err != nil {
 			util.HandleError(err, "Unable to parse flag")
 		}
 
+		projectId, err := cmd.Flags().GetString("projectId")
```

Comment:
use dash case `project-id`

Reply:
Yeah I agree, but we are using camelCase for the project ID elsewhere, so I wanted to keep things consistent by using camelCase elsewhere. Let me know if you still think we should use dash-case.